libcxx/test/std/library/description/conventions/customization.point.object/niebloid.compile.pass.cpp

```diff
@@ -83,6 +83,10 @@ static_assert(test(std::ranges::find_end, a, a));
 static_assert(test(std::ranges::find_first_of, a, a));
```

Comment:
Can you also take a look at various `robust` tests and add the new algorithms where it makes sense (e.g. `libcxx/test/std/algorithms/ranges_robust_against_omitting_invoke.pass.cpp` looks relevant)?

Reply:
I'm a bit hesitant to use that test file, so `left_folds.pass.cpp` does checks to ensure invocable robustness. I'll my concerns in the new year, when I tackle the other folds.